cpp/include/cuml/fil/fil.h

```diff
@@ -85,6 +85,11 @@ struct treelite_params_t {
   // suggested values (if nonzero) are from 2 to 7
   // if zero, launches ceildiv(num_rows, NITEMS) blocks
   int blocks_per_sm;
+  // threads_per_tree determines how many threads work on a single tree at once inside a block
+  int threads_per_tree;
```

Comment:
Isn't it limited to powers of 2?

Reply:
will fix after I check for perf regressions on threads_per_tree==1 case